website/source/docs/providers/google/d/datasource_compute_network.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "google"
+page_title: "Google: google_compute_network"
+sidebar_current: "docs-google-datasource-compute-network"
+description: |-
+  Get a network within GCE.
+---
+
+# google\_compute\_network
+
+Get a network within GCE from its name.
+
+## Example Usage
+
+```tf
+datasource "google_compute_network" "my-network" {
+  name = "default-us-east1"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the network.
+    
+
+- - -
+
+* `project` - (Optional) The project in which the resource belongs. If it
+    is not provided, the provider project is used.
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following attributes are exported:
+
+* `network` - The network name or resource link to the parent
+    network of this network. 
+
+* `description` - Description of this network.
+
+* `ip_v4_range` - (DEPRECATED) The IPv4 address range that machines in this network
```

Comment:
If this is deprecated do you actually need to be exporting it?

(also the field name you used in the data source was ipv4_range)